.travis.yml

```diff
@@ -13,8 +13,9 @@ before_install:
 install:
   # installing qiita environment
   - conda config --add channels anaconda
-  - travis_retry conda create -q --yes -n qiita python=3.6 pip libgfortran numpy nginx
+  - travis_retry conda create -q --yes -n qiita python=3.6 pip libgfortran numpy nginx 
   - source activate qiita
+  - pip install cython
```

Comment:
Suggest adding this to the conda create line 16

Reply:
Done.